model/training/data/data_loader.py

```diff
@@ -0,0 +1,74 @@
+import json
+from cv2 import imread, IMREAD_GRAYSCALE
+
+
+class DataLoader:
```

Comment:
Czy na tą chwilę to jest gdzieś wykorzystywane? Czy to raczej zostało napisane dla trenowania sieci? Bo jeśli tak, to mmdetection chyba sam sobie radzi z tym

Reply:
już niepotrzebne, racja, to było przed (zamiast) convertera do COCO